### DISCUSSION TITLE
New icons

### DISCUSSION THREAD

**Discussion by adamsthws** [chrede88/L1nkr#162] [Ideas]
Hello, 

Thankyou for amking L1nkr, it's excellent!

I would like to create a new icon: 'PGP key'. Would you be interested in meging this if I create a PR if the new icon matches the style of the exising icons?

Kind regards,
Adam

**Comment by chrede88** [comment]
Sure, absolutely :+1:

**Reply by adamsthws** [reply]
Thankyou for merging my [PR](https://github.com/chrede88/L1nkr/pull/163)... I've also created icons for 'Certifications' and 'SSH-Keys', would you like me to create a PR for either (or both) of these too? - You can see examples at: [admatthews.co.uk](https://admatthews.co.uk/)

Thanks, Adam

**Reply by chrede88** [reply]
Sure, that would be great. You can just add both to a single PR☺️
